spec/lib/data-sf/index.spec.js

```diff
@@ -0,0 +1,19 @@
+import { getParklets, parseParklet } from '../../../src/lib/data-sf';
+
+describe('Data SF', () => {
+  describe('#getParklets()', () => {
+    test('Should fetch blob of parklet data', () => (
+      getParklets().then(data => expect(data.length).toBeGreaterThan(0))
```

Comment:
rather than testing the length, you should check the status code to see if it was a good response.

Reply:
Hmm perhaps refactor to also return statuscode and error message (if it exists). This way the error doesn't get swallowed up. Then you'll have two branches of logic to write some meaningful tests for that will give insight into what's going on. I'd recommend having a dir (`__mocks__`) to have your mocks and stubs so that you can test both paths as a unit test then hit the live API as a psuedo integration test.